src/forms/FormFieldText/FormFieldText.styled.tsx

```diff
@@ -26,14 +26,16 @@ export const StyledTextField = styled(({ fieldSize, ...rest }) => (
 	}
   }
 
-  .MuiOutlinedInput-multiline, input.MuiOutlinedInput-input {
-    padding: ${theme.fieldSpecs.inputText.padding};
+  .MuiInputBase-multiline {
+	padding: 0px;

```

Comment:
I just noticed that in some places the code is spaced using tabulations and in others uses spaces. I think we should use tabs for all the file since that's our linter configuration and when we run it we should not have any issues.